openhands-tools/pyproject.toml

```diff
@@ -13,6 +13,7 @@ dependencies = [
     "pydantic>=2.11.7",
     "browser-use>=0.7.7",
     "func-timeout>=4.3.5",
+    "tom-swe @ git+https://github.com/all-hands-ai/ToM-SWE.git@main",
```

Comment:
Could you publish this via PyPI? if we point to github branch, it is going to cause issues for building executable binaries for CLI